app/src/etc/supervisord.conf

```diff
@@ -46,7 +46,17 @@ command = node NodeJSServer.js
 directory=/home/bdt/map/src/map_nodejs_server/
 stdout_logfile=/var/log/NodeJSServer.log
 stderr_logfile=/var/log/NodeJSServer.log
-autostart=false                ; start at supervisord start (default: true)
+autostart=true                 ; start at supervisord start (default: true)
 autorestart=false              ; restart at unexpected quit (default: true)
 stopasgroup=true               ; stop all subprocesses together
 killasgroup=true               ; kill all subprocesses together
+
+[program:startup]
+command = /bin/bash /home/bdt/map/src/map_scripts/startup_after_shutdown.sh
```

Comment:
iiuic the script is in the (private) bdt repo, but it's needed for this (public) tacstack repo.
generally, the public repo should be self-sufficient, excluding only specific internal data that a public user could plug in from their own private sources.

does that make more sense?